prow/oss/terraform/main.tf

```diff
@@ -42,4 +42,7 @@ module "alert" {
         "sinker" = {"namespace": "default"}
         "tide" = {"namespace": "default"}
     }
+    blackbox_probers = [
+        "oss-prow.knative.dev"
```

Comment:
I see an existing uptime check for `testgrid.k8s.io`, was that manually added for testing?

Reply:
yep